### PR TITLE
New repository

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
@philkark please merge https://github.com/philkark/squirrel_motion_planner/pull/2

**Comment by NadiaHG** [comment]
@philkark I am getting the following error when I compile the repository:
```
/tmp/squirrel_motion_planner_indigo_prtestbuild/src/squirrel_motion_planner/squirrel_8dof_planner/src/squirrel_8dof_planner.cpp: In member function 'bool SquirrelMotionPlanner::Planner::findTrajectoryEndEffector(const std::vector<double>&)':
/tmp/squirrel_motion_planner_indigo_prtestbuild/src/squirrel_motion_planner/squirrel_8dof_planner/src/squirrel_8dof_planner.cpp:560:7: error: cannot convert 'SquirrelMotionPlanner::Planner::serviceCallGetOctomap' from type 'bool (SquirrelMotionPlanner::Planner::)()' to type 'bool'
   if(!serviceCallGetOctomap)
       ^
/tmp/squirrel_motion_planner_indigo_prtestbuild/src/squirrel_motion_planner/squirrel_8dof_planner/src/squirrel_8dof_planner.cpp:560:7: error: in argument to unary !
/tmp/squirrel_motion_planner_indigo_prtestbuild/src/squirrel_motion_planner/squirrel_8dof_planner/src/squirrel_8dof_planner.cpp: In member function 'bool SquirrelMotionPlanner::Planner::findTrajectoryPose(const std::vector<double>&)':
/tmp/squirrel_motion_planner_indigo_prtestbuild/src/squirrel_motion_planner/squirrel_8dof_planner/src/squirrel_8dof_planner.cpp:571:7: error: cannot convert 'SquirrelMotionPlanner::Planner::serviceCallGetOctomap' from type 'bool (SquirrelMotionPlanner::Planner::)()' to type 'bool'
   if(!serviceCallGetOctomap)
       ^
/tmp/squirrel_motion_planner_indigo_prtestbuild/src/squirrel_motion_planner/squirrel_8dof_planner/src/squirrel_8dof_planner.cpp:571:7: error: in argument to unary !
make[2]: *** [squirrel_motion_planner/squirrel_8dof_planner/CMakeFiles/squirrel_8dof_planner_node.dir/src/squirrel_8dof_planner.cpp.o] Error 1
make[1]: *** [squirrel_motion_planner/squirrel_8dof_planner/CMakeFiles/squirrel_8dof_planner_node.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by NadiaHG** [comment]
@philkark cloud you please readd the travis configuration?

There are declared two dependencies to packages that don't exist:
- kuka_lbr_moveit_config
- birrt_star

please remove these dependencies

**Comment by NadiaHG** [comment]
@philkark Buildbot says: 
`Could not find planning_server in keys.`
I think it is because the run_depend is not defined in the metapackage

File: birrt_star/robot_motion_control/kuka_motion_control/package.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_depend>kdl_parser</build_depend>
+  <build_depend>moveit_core</build_depend>
+  <build_depend>moveit_ros_planning</build_depend>
+  <build_depend>tf</build_depend>
+  <build_depend>srdfdom</build_depend>
```
Comment (NadiaHG) [inline birrt_star/robot_motion_control/kuka_motion_control/package.xml]: @philkark should also be listed as run_depend

**Comment by NadiaHG** [comment]
@philkark updates here?

**Comment by NadiaHG** [comment]
@philkark please update the repository

**Comment by NadiaHG** [comment]
@philkark please remove the `birrt_star/robot_motion_planning/planning_server/CATKIN_IGNORE` file.

Are there any reason to don't clone directly `https://github.com/burgetf/robot_motion_planning` and make a copy??

**Comment by NadiaHG** [comment]
@philkark why are you removing the travis configuration??

**Comment by ipa-rmb** [comment]
@philkark indeed, please do not remove the travis configuration. If software does not build on Travis, it is very likely it will not work on any other computer without manual configuration steps. The Travis test must be passed before we can accept pull requests. There is a merge conflict in the current status, in addition.

**Comment by NadiaHG** [comment]
@philkark please merge https://github.com/philkark/squirrel_motion_planner/pull/3 to solve the merge conflict and re-add the travis configuration

**Comment by NadiaHG** [comment]
@philkark please merge  https://github.com/philkark/squirrel_motion_planner/pull/4 to solve the conflicts

**Comment by NadiaHG** [comment]
@philkark please remove the file  `birrt_star/robot_motion_planning/endeffector_target_tracking/CATKIN_IGNORE`

File: squirrel_8dof_planner/include/squirrel_8dof_planner/squirrel_8dof_planner.h [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+#include <ros/ros.h>
+#include <ros/package.h>
+
+#include <nav_msgs/OccupancyGrid.h>
+#include <nav_msgs/Path.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/PoseWithCovarianceStamped.h>
+#include <squirrel_motion_planner_msgs/PlanEndEffector.h>
+#include <squirrel_motion_planner_msgs/PlanPose.h>
+#include <squirrel_motion_planner_msgs/UnfoldArm.h>
```
Comment (NadiaHG) [inline squirrel_8dof_planner/include/squirrel_8dof_planner/squirrel_8dof_planner.h:20]: file not found, squirrel_motion_planner_msgs has to be updated??

**Comment by NadiaHG** [comment]
@philkark I removed the CATKIN_IGNORE for `endeffector_target_tracking` and the folder old (where all packages had a CATKIN_IGNORE file) see https://github.com/squirrel-project/squirrel_motion_planner/pull/9

I also had to update the travis configuration to don't test the install work space, there is a relative path include https://github.com/ipa-nhg/squirrel_motion_planner/blob/77181341560cc79528eedb0398284919e706ca26/birrt_star/robot_motion_control/kuka_motion_control/include/kuka_motion_control/control_laws.h#L32 and this is not allow, it would be nice if this can be fixed.

Let me know if the remove of the CATKIN_IGNORE's is ok for you to merge the pull request.

**Pull request closed without merging**